docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -23,166 +23,187 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 - `BalanceChange`: Any changes to the balance
-- `Authorization`: How the zkApp is authorized; can be either a proof (corresponding to the verification key on the account) or a signature.
+- `Authorization`: How the zkApp is authorized; must be a proof (corresponding to the verification key on the account), a signature, or none. See [Authorizations](/zkapps/o1js/interact-with-mina#authorizations).
 
-These AccountUpdate components are available to use but are not covered in this tutorial:
+Other `AccountUpdate` components are available to use, but are not covered in this tutorial:
 
 - `MayUseToken`: Whether the zkApp has permissions to manipulate its token.
-- `Layout`: Allows for assertions about the structure of an AccountUpdate.
+- `Layout`: Allows for assertions about the structure of an `AccountUpdate`.
 
-## AccountUpdates for a non-user-upgradable zkApp
+## Account updates for a non-upgradable zkApp
 
-Now, you can start building an example zkApp to explore permissions, preconditions, and composability with AccountUpdates.
+If the verification key cannot be changed, the zkApp smart contract is considered non-upgradeable. The `setVerificationKey` permission sets the ability to change the verification key of the account.
 
-To visualize transactions, use the library `mina-transaction-visualizer`. To use this library in your own zkApp, install with npm with `npm install mina-transaction-visualizer --save`.
+Now, you can start building an example zkApp to explore permissions, preconditions, and composability with `AccountUpdates`.
 
-You can find the full [source code for this Account Updates tutorial](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/10-account-updates) in the `examples` directory within the Mina docs repo.
+### Visualize transactions
 
-### Smart Contracts
+To visualize transactions, use the `mina-transaction-visualizer` library. Install this library to use in your own zkApp:
 
-In this tutorial, you build two smart contracts to use in your example:
+```sh
+npm install mina-transaction-visualizer --save
+```
 
-First, [ProofsOnlyZkApp.ts](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts).
+## Smart Contracts
 
-Configure this zkApp to be modifiable only by using proofs. For this example, the zkApp is not upgradable after it is deployed. This means that while the zkApp developer owns the private key to initially deploy the zkApp, after its first deployment, the zkApp requires proof authorization and consequently can only be updated by transactions that fulfill the zkApp's smart contract logic. The private key is no longer useful for anything.
+In this tutorial, you build two smart contracts: 
 
-This zkApp also has methods that call other methods, so you can explore how that impacts a transaction's AccountUpdates.
+- `ProofsOnlyZkApp`: Non-upgradeable proof only
+- `SecondaryZkApp`: the other zkApp
```

Comment:
not sure what to call "the other zkApp" who has a suggestion?

Reply:
We could call it the "childZkApp" though I'm not sure if that fits perfectly because it can also be used standalone.